standard/classes.md

```diff
@@ -3426,6 +3426,7 @@ For a ref-valued property the *ref_get_accessor_declaration* consists optional a
 
 The use of *accessor_modifier*s is governed by the following restrictions:
 
+- An *accessor_modifier* shall not be used in an explicit interface member implementation.
```

Comment:
This restores a bullet that I believe still holds and I didn’t spot covered elsewhere.

Reply:
This one?

> It is a compile-time error for an explicit interface member implementation to include any modifiers ([§15.6](https://github.com/dotnet/csharpstandard/blob/draft-v8/standard/classes.md#156-methods)) other than extern or async.

(Just to check I've got the right one, and to help others who would look...)